.github/workflows/wai-deploy.yml

```diff
@@ -10,8 +10,21 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: actions/setup-node@v1
-      - run: git config user.name "${{ secrets.USER_NAME }}"
-      - run: git config user.email "${{ secrets.USER_EMAIL }}"
-      - run: npm install
-      - run: npm run build:wai
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - uses: actions/cache@v3
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+      - name: Configure git
+        run: |
+          git config --global url."https://${{ secrets.WAI_GIT_NAME }}:${{ secrets.WAI_GIT_ACCESS_TOKEN }}@github.com".insteadOf "https://github.com"
+          git config --global user.name "${{ secrets.WAI_GIT_NAME }}"
+          git config --global user.email "${{ secrets.WAI_GIT_EMAIL }}"
```

Comment:
These are new secrets. They're currently set up with my personal credentials. I'll ask W3C what their policy on credentials is. I'd prefer to get an account set up by the W3C for this.